### PR TITLE
Output initial request helper method for LROs

### PR DESCRIPTION
**Pull request by catalinaperalta**
Add a wrapper similar to the other non-LRO operations for the LRO endpoints.

### CODE REVIEW

**Comment by catalinaperalta** [comment]
NOTE: response handlers and final response updates will be made in a separate PR

File: src/transform/namer.ts [hunk 1]
```diff
@@ -103,7 +107,7 @@ export async function namer(session: Session<CodeModel>) {
       details.name = getEscapedReservedName(capitalizeAcronyms(pascalCase(details.name)), 'Method');
       // add the client name to the operation as it's needed all over the place
       details.clientName = groupDetails.clientName;
-      if (isLROOperation(op)) {
+      if (isLROOperation(op) && !isDataPlane) {
```
Comment (jhendrixMSFT) [inline src/transform/namer.ts:110]: I don't think we should be adding the prefix, it just adds complexity.  Instead, for non-data-plane we create the additional `Begin*` method.

**Comment by jhendrixMSFT** [comment]
I think you'll want to rebase on https://github.com/Azure/autorest.go/pull/487 once it's merged so that we can see the effects on data-plane pollers.

File: src/transform/namer.ts (hunk 1 above)
Comment (catalinaperalta) [reply src/transform/namer.ts]: Changed it, although I'm not sure I like the lingering MethodPrefix. I could remove it completely and we can add it back in if necessary

**Comment by jhendrixMSFT** [comment]
FYI I rebased and regenerated.

File: src/transform/namer.ts (hunk 1 above)
Comment (jhendrixMSFT) [reply src/transform/namer.ts]: I went ahead and removed `methodPrefix`

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**